CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [4.8.0] - 2022-03-23
```

Comment:
@kyle-bowden apologies, merged another change ahead of this so needs rebasing / incrementing version number

Reply:
and, because I've made a change to the backstop config you probably need to remove your backstop image changes then regenerate them